File: 백준/Bronze/2775. 부녀회장이 될테야/부녀회장이 될테야.java
import java.util.Scanner;

class Count{
   
    public int living(int k, int n){
        
        int live[][] = new int [k+1][n+1];
        
        for(int i = 1; i <= n; i++){
            live[0][i] = i;
        }
        
        for(int j = 1; j <= k; j++){
            for(int x = 1; x <= n; x ++){
                if(x != 1){
                    live[j][x] = live[j][x-1] + live[j-1][x];
                }
                else{
                    live[j][x] = 1;
                }
                
            }
        }
        
         return live[k][n];
        
    }
    
}

public class Main{
    public static void main(String [] args){
        
        Scanner t = new Scanner(System.in);
        
        int test,k,n,result;
        test = t.nextInt();
        
        Count count = new Count();
        
        for(int i = 0;i < test; i++){
            k = t.nextInt();
            n = t.nextInt();
            result = count.living(k,n);
            System.out.println(result);
        }
        
        t.close();
    
    
    }


}

File: 백준/Bronze/2775. 부녀회장이 될테야/부녀회장이 될테야.cc
#include <stdio.h>

int living(int k1, int n1);


int main(void){
    
    int test, k, n, people;
   
    // printf("test case 의 횟수를 입력하시오:");
    scanf("%d",&test);
    
    for(int i = 0; i < test; i++){
    	
    	
        // printf("거주할 층과 호수를 입력하시오\n");
        scanf("%d",&k); // 층 입력
        scanf("%d",&n); // 호수 입력 
        people = living(k,n);
        printf("%d\n",people);
        
       
   
}

 return 0;
}

int living(int k1, int n1){

	int live[k1 +1][n1 +1]; 
	
	for(int i = 1; i <= n1; i++){
		live[0][i] = i;
	}
	
	
	for(int i = 1; i <= k1; i++){
		for(int t = 1 ; t <=n1; t++){
			
		if(t != 1)
			live[i][t] = live[i][t-1] + live[i-1][t];
		
		else
			live[i][t] = 1;
		
	
	}
	
	
	}
	return live[k1][n1];
}



